misk/src/main/kotlin/misk/web/interceptors/ConcurrencyLimitsInterceptor.kt

```diff
@@ -59,6 +69,17 @@ internal class ConcurrencyLimitsInterceptor internal constructor(
     }
   }
 
+  private fun logShedRequest() {
+    val nowMs = clock.millis()
+    val durationSinceLastErrorMs = nowMs - lastErrorLoggedAtMs
```

Comment:
Do we want to keep track and log how many have happened since the last log statement? Might be useful. But I guess you are right in that we can rely on metrics.

Reply:
Yeah I think that’s easier in metrics anyway.